web/src/message_list_view.ts

```diff
@@ -85,6 +104,7 @@ function analyze_edit_history(message, last_edit_timestr) {
             }
 
             if (edit_history_event.prev_topic) {
+                assert(edit_history_event.topic !== undefined);
```

Comment:
Possibly this assert could be removed if we tightened the type on edit history elements such that a `prev_topic` being present means a `topic` element is.